Podfile.lock

```diff
@@ -73,7 +73,7 @@ PODS:
   - GTMSessionFetcher/Core (1.5.0)
   - GTMSessionFetcher/Full (1.5.0):
     - GTMSessionFetcher/Core (= 1.5.0)
-  - Gutenberg (1.46.0):
```

Comment:
Just came here from https://github.com/WordPress/gutenberg/pull/28766 and noticed that this is going down a version (1.46.0 to 1.45.0), could your Gutenberg Mobile or Gutenberg branches need updating?